ipatests/test_xmlrpc/test_certmap_plugin.py

```diff
@@ -122,6 +139,25 @@ def test_delete(self, certmap_rule):
         certmap_rule.ensure_exists()
         certmap_rule.delete()
 
+    @pytest.mark.parametrize(
```

Comment:
With this parametrize annotation, the test will be run with all combinations of missing params. When all params except ipacertmapmaprule are provided, the test won't fail while the code expects it to fail.
The test can probably be simplified (no need to test all combinations of parameters, we are simply interested in checking that altsecurityidentites in the maprule triggers a validation error).

Reply:
Yes, I removed all that complexity. A new test is running -- I need to understand why ensure_missing() tracker calls are causing exceptions as well.